tests/chapter2-exercise-emoji-tests.md

```diff
@@ -49,6 +49,20 @@ Testing boilerplate; hiding create_rectangle because of scrollbar tests:
     >>> _ = wbemocks.socket.patch().start()
     >>> _ = wbemocks.ssl.patch().start()
     >>> _ = wbemocks.patch_canvas()
```

Comment:
This is good but this code should live in the `wbemocks.py` file, not here. Here's the code snippet from the TA meeting:

```
    
TK_INITIALIZED = False

class SilentTk:
    def __init__(self, *args, **kwargs):
        global TK_INITIALIZED
        TK_INITIALIZED = True

    def bind(self, event, callback):
        pass

tkinter.Tk = SilentTk
    
class PhotoImage:
    DO_NOT_GC = weakref.WeakKeyDictionary()

    def __init__(self, file, *, secret_width=None, secret_height=None):
        assert TK_INITIALIZED, "You cannot create a PhotoImage until after a Tk is constructed"
        # ...
```